src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="index.html">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+               class="nav__link"
+               href="index.html"
+               data-qa="hover"
+            >
```

Comment:
Should be only 2 spaces for attributes
```suggestion
            <a
              class="nav__link"
              href="index.html"
              data-qa="hover"
            >
```